configuration/concepts/hospitals.csv

```diff
@@ -1,10 +1,10 @@
-4b9b9184-6aa8-4875-ab10-f22092a0075b,,,St Jean Limbe Hospital,Hopital St Jean Limbe,St Jean Limbe Hospital,Hopital St Jean Limbe,,,Misc,N/A,,,,,,,,,,,
-1ff3db17-7a6f-40bf-8062-a912773e905c,,,Good Samaritain Limbe Hospital,Hopital Bon Samaritain Limbe,Good Samaritain Limbe Hospital,Hopital Bon Samaritain Limbe,,,Misc,N/A,,,,,,,,,,,
-b0b1c920-02ba-47e7-93e5-9922e888aac0,,,Justinien Cap Haitien Hospital,Hopital Justinien Cap Haitien,Justinien Cap Haitien Hospital,Hopital Justinien Cap Haitien,,,Misc,N/A,,,,,,,,,,,
-696114ce-8c7b-4018-a8d5-ee2e65d323d3,,,Sacred Heart Milot Hospital,Hopital Sacré Coeur Milot,Sacred Heart Milot Hospital,Hopital Sacré Coeur Milot,,,Misc,N/A,,,,,,,,,,,
-50f4de56-b5b6-4939-aa73-7d7f5a5f8961,,,HBCH Hospital,Hopital HBCH,HBCH Hospital,Hopital HBCH,,,Misc,N/A,,,,,,,,,,,
-57195bab-5935-4da4-b5de-e9ee1dba162e,,,Gable Hospital,Hopital de Pignon,Gable Hospital,Hopital de Pignon,,,Misc,N/A,,,,,,,,,,,
-1c61317b-f7cf-4709-829c-76abb6f6501b,,,Borgne Hospital,Hopital de Borgne,Borgne Hospital,Hopital de Borgne,,,Misc,N/A,,,,,,,,,,,
-6d1450bb-0109-456b-82f9-e495bcb5c959,,,Trou du Nord Hospital,Hopital de Trou du Nord,Trou du Nord Hospital,Hopital de Trou du Nord,,,Misc,N/A,,,,,,,,,,,
-ebf58d81-6ca4-4f2b-9287-cffa5cb67441,,,Other Health Centre,Autre centre de santé,Other Health Centre,Autre centre de santé,,,Misc,Text,,,,,,,,,,,
+4b9b9184-6aa8-4875-ab10-f22092a0075b,,,Hopital St Jean Limbe,Hopital St Jean Limbe,Hopital St Jean Limbe,Hopital St Jean Limbe,,,Misc,N/A,,,,,,,,,,,
+1ff3db17-7a6f-40bf-8062-a912773e905c,,,Hopital Bon Samaritain Limbe,Hopital Bon Samaritain Limbe,Hopital Bon Samaritain Limbe,Hopital Bon Samaritain Limbe,,,Misc,N/A,,,,,,,,,,,
+b0b1c920-02ba-47e7-93e5-9922e888aac0,,,Hopital Justinien Cap Haitien,Hopital Justinien Cap Haitien,Hopital Justinien Cap Haitien,Hopital Justinien Cap Haitien,,,Misc,N/A,,,,,,,,,,,
+696114ce-8c7b-4018-a8d5-ee2e65d323d3,,,Hopital Sacré Coeur Milot,Hopital Sacré Coeur Milot,Hopital Sacré Coeur Milot,Hopital Sacré Coeur Milot,,,Misc,N/A,,,,,,,,,,,
+50f4de56-b5b6-4939-aa73-7d7f5a5f8961,,,Hopital HBCH,Hopital HBCH,Hopital HBCH,Hopital HBCH,,,Misc,N/A,,,,,,,,,,,
+57195bab-5935-4da4-b5de-e9ee1dba162e,,,Hopital de Pignon,Hopital de Pignon,Hopital de Pignon,Hopital de Pignon,,,Misc,N/A,,,,,,,,,,,
+1c61317b-f7cf-4709-829c-76abb6f6501b,,,Hopital de Borgne,Hopital de Borgne,Hopital de Borgne,Hopital de Borgne,,,Misc,N/A,,,,,,,,,,,
+6d1450bb-0109-456b-82f9-e495bcb5c959,,,Hopital de Trou du Nord,Hopital de Trou du Nord,Hopital de Trou du Nord,Hopital de Trou du Nord,,,Misc,N/A,,,,,,,,,,,
+ebf58d81-6ca4-4f2b-9287-cffa5cb67441,,,Autre centre de santé,Autre centre de santé,Autre centre de santé,Autre centre de santé,,,Misc,Text,,,,,,,,,,,
```

Comment:
@nuhuhmutebi this is not a hospital. Meanwhile its more of a description so we keep it the way it was

Reply:
Refer to this https://mekomsolutions.slack.com/archives/G421UNF5L/p1636637435092700?thread_ts=1636532774.043600&cid=G421UNF5L